### DISCUSSION TITLE
Subscriptions Question

### DISCUSSION THREAD

**Discussion by daudcanugerah** [nats-io/nats-server#3731] [Q&A]
<img width="1373" alt="Screen Shot 2022-12-21 at 12 59 13" src="https://user-images.githubusercontent.com/34634382/208832515-2c9eba71-2dbe-4a2e-b0bd-b5185e1a1bd8.png">

does the value of gnatsd_varz_subscriptions mean the total subscribers currently running . if true, how do you find out which subscriber has a problem. after reaching 2m nats server becomes slow.
i am using one nats server with version 2.8.4.
I'm not sure if this issue (https://github.com/nats-io/nats-server/issues/2170) is related. Thank you.

**Comment by ripienaar** [comment]
In a cluster that number is a bit inaccurate but it’s a good indication of trends. 

You can try using the nats command line to report connections and see if you have some with many subscriptions and also can then look at specific clients. See the sun commands of “nats server report” and “nats server request”

**Reply by daudcanugerah** [reply]
run "nats server report" get erorr nats: error: did not get results from any servers
i can't find "nats server request" in subcommand
<img width="406" alt="Screen Shot 2022-12-21 at 13 19 22" src="https://user-images.githubusercontent.com/34634382/208835417-01951ac7-4ba5-4d12-8f86-bb80abe6d913.png">

**Reply by ripienaar** [reply]
You need to have a system account configured to access this data. Double check your command for “nats server request”

**Reply by daudcanugerah** [reply]
after analyzing the command above.
what is meant by _inbox.xxx in the subscription_list?
the application that I made only subscribes to the jetstream with the subject "crawler.*.*.*" and the nats subject "worker.signal"

```json
    "subscriptions_list": [
      "_INBOX.kD89nr.*",
      "_INBOX.1o2OA7.*",
      "_INBOX.e0Z19L.*",
      "_INBOX.ZpMu5N.*",
      "_INBOX.wEQbGGE4NGyKFMQQJe5Gtm",
      "_INBOX.wEQbGGE4NGyKFMQQJe5Gw2",
      "_INBOX.wEQbGGE4NGyKFMQQJe5GyI",
      "_INBOX.PIkZpg.*",
      "_INBOX.XXpBFx.*",
      "worker.signal",
      "_INBOX.wEQbGGE4NGyKFMQQJe5Gn0.*",
      "_INBOX.wEQbGGE4NGyKFMQQJe5GpG",
      "_INBOX.wEQbGGE4NGyKFMQQJe5GrW"
    ],

```
